internal/gatewayapi/route.go

```diff
@@ -252,7 +255,11 @@ func (t *Translator) processHTTPRouteRules(httpRoute *HTTPRouteContext, parentRe
 
 		// process each ir route
 		for _, irRoute := range ruleRoutes {
+			if len(backendCustomRefs) > 0 {
+				irRoute.ExtensionRefs = append(irRoute.ExtensionRefs, backendCustomRefs...)
+			}
 			destination := &ir.RouteDestination{
+				Name:     destName,
```

Comment:
curious why its needed
its set below

```
		default:
				destination.Name = destName
				destination.Settings = allDs
				irRoute.Destination = destination
```

Reply:
just want to make sure the destination to have a name